Packs/Arcanna/Integrations/ArcannaAI/ArcannaAI.yml

```diff
@@ -1,181 +1,521 @@
Cx1+hQt08SAeLkymSsdM4ATgQjU1GuylhQEXDeqigDDgokFdFOiewUWp76rsld7hwtoSeKUGe1lhwEWhxQPpC73DBbooLwy4KKOLVZzPDS4s6aJig/3fjHBhKKMRXWgLAy4a1YVy9wwuSlR7iwqjcGEmlRvs5YQBF63qQlcYcNGsLlS7Z3Chr4td63w8XJiIgQZ7KWHAhfrXczd3RnDRty4Uu2dwoV3+b5PiOc1wUT2rxSu8wwW6KCgMuFDWxap8Whtc1NWFoY7Zx4xwUTWjTV1otdvhom1daAkDLtrWhVb3DC5U54O+wKmtcFEtthrsBYQBF63rQkcYcGFGF34el59ZRx9Vrnq4qBT9jtm8/jvhWdZJ/fTgIjPaDfZpfDROLb6iMOCivi7Gr9ZGgslY4KIdXQS+UcJ/d1PHsNFkhosKUW2wP7mlgcrY4aL8Erhmg/0pc9tcRRhwUVUXq1SZssNFG7oI6pitYvXrCBeFo9gxCzTRFlBjSLfb4ULpggfoIrgo2KfiwoCLarqIIC7k0+CiaPTeKBGzGBVQYqxwUTB6HbOo1bK9tDDgQmX+J6yL8sKAixZ0UV4YcKGhi1laFyofCRep0WuwR3+5V7WzhYvo6DXYoweozRUVBlxU0UVCkegVYIMLY7pIKFxGlU+Fi4QlcL0Ge8JHh/yIeIaLAlFssHsdLuSEARc1dJFUubiSwoALaV0EPZI6KnEh1m6HC+lLPNXkQqzdDhfCV3h1VbmQEgZc1NBFUh2gOvrBRWB0H0n1alxMcKEa3SfYvRpxQsKAixq6cE7voye4aFYXKZ8f/HD8AhfWdBG+Rfyqh5yHC7Wob3ih0mcXFAZcCH7lIt4oET0NXtXPHi7q6yJ+IPGuqDDgQq6kj9oi3uvMRnKqI7jQWQIYFZ0U93KbHS4a1UWkMCI3Ap3holVdxB0mdiPQDS6s6CJ6i3jZ55b/zQgX4im2RXzwSLJEv5Ugu90OF1KXNOGNErNK4SIiDLiopovQEmNL2WV8ggvhFN0iflXCIrvdDhcyU7xFy06JWOQKAy5q6uKNw0245JQRBlxU1cXbF/ubhc8t4x3eHi4EU+Md7PNXjlqnCv8KXMh9yfZcSz24h9s6Vflf4MKILn6NJv+isS3zVOufgQsruvhz9Hlcl2UZx1nmvWgzXAhl7eIi5s6t4EJmzr9NRrmY4QJdyAoDLvKXCCezXIxwUU+7q1ksMtrtcJFtXbu6yBAGXOQOI4Z1kSEMuMgdRrxlLpK7Z3CReQkX01gk19Jw8Z6lQ130MveuyUWPukgWBlzkcTFb5yJRg3CRxcVuHovEXiBcZHFhXxeJwoCLHC4a0EWiMOAih4sWdJG2jjvDxZ/ELw1uUxNczO1Pv9tavxid61UYE1wkrwA1oosUYWwOLpKv3toIFgndswUu0lcGW9FFQrt9hovkAqMZXSQIY4KL5IGkHV1E107mkK/7+86t5SFYcoz0cJE8DPuWuIgThj3kKz8/snWqi0gXerhIrjCmtriIcaHBgrqZ55ZH11qC/7V9gotU3S7NYeH81nDhVH1fFN/sV0pqkDTZI25j37XNO9crGDZ/OmBgn8a5VyyCwDD6ixIL+0A/2SGvzUEk7F/brP7QyMT7BPy3pfvaLhb3f+3bn2IsZkVo5D1WX0/299k1nenrhc/N8NzbyvsQv7h82zi51vMVGaY9aOc93NO6fXZF+1R89a8ZB94OF29V2rp83Djx/+3a63arIBCGYReu9s8m6Nz/zW4P1aLBaBRsYN7vChp8Ogf0qyon/k/7l8FP+ygXw//W15Dvqrz8/LQs/taPc0FwQXBBcEGyT4MLEojBBcEFwQXBBYnuwnEIBBfkUBwuSNCF5RBIwIVwCOQpUsmDUyABFzWnQFapcUE2XHCxRdYxuCAbLrjAIOu0nQsuMMg6rnPBBQZ5WlN7FywkZL2O9C4YPMkyDS5IcB3pXbCQkPXY2btg8CTL2NEFgyfx85DRBQMGWY2dg4uWoyBe2h8X3HiS1XgxuODTHLIaL0YXDBhkOV6MLhgwyHK8GF0wYJDleDG64AaDzKnl1wUDBpliPBc0EjLFeS7YVIm/pc4uaCTEbyOTC961E7+NTC7YSIjfRmYXNBLitZHZBRsJ6WNXLmgkpJoutXwXLYdCxncjCxdcYZDqYZ9dMHmSRp5dMHkSF3AhDefC1BlwwZ0nU2fIBZOn9qlTwi5YVSkXIRcUDMpF0AUFQ3PMpgsKhuZyYbddUDAoFyEXFAzKRdAFBYNlJOSC1+0sI0EXXHpSLkIuKBgqU8ueC16raozbdcF3GOp31LALy66qfEcNu2BXVT90hl3wgY72oXPDBZ1EexcJu6CTKB86t1zQSZR3kS0XdBLdXWTLBZ1E8y7ywgW3W0rSyHsuLO9JFHeRbRfiGDEUsHDyrgtGDL3DxUsXjBhqh4vXLvgUQ+XNxb4LZk+dM+eeC2ZPnTPnrgu+9lQ5c+67YClRymLPBUtJoTFyzQWvVrVtqAddAEMjiwMugKGQxREXwCgstY3jAhjqWBxzAQxdTeSwC2AoY3HUBTB0sTjsQlrOtIAYie0CGAWklfgueLuaex7HWbzjQhzfY2TNwkkaF2KZPgu/tjjngterCibOUy6kZcgofbQ45YKPPrPsIU5Su6CXZNhDrNzggoW18B5y1gV7SU5prNzlgvGz7GJx3oVYpoxyi8UFF0wZORQLd/rpnnfRLSbI+GgV5sKzveKC+bPIFnLdBa/SCrrJiuoCGUVtIRFdsLMWqCKKi04GNaMsFZFc0E2KmSsiu2A3+YwdxMV6nNFc9FegDBp/e19h4z3MiC76dtJAI/tSkcBFVzRa+skfoGht5OcY2wU07kdhbPyHmMDFSIOGcstMEb9SpHQxzBqG3TXxTmpcsqeXzsVog7qRqE4kNJHexdBTOhxUjohVoiNhkz+19C5mHR0PfFzw0IG4Q8S9LjwgfQw5mrY/L3v3Y/oP1Aw0t7bBHQQAAAAASUVORK5CYII=
```

Comment:
Please remove the image from the yml file. 
You can run demisto-sdk format and validate on the file to make sure it in the right format.
https://github.com/demisto/demisto-sdk/blob/master/demisto_sdk/commands/format/README.md